support-figma/extended-layout-plugin/src/localization-module.ts

```diff
@@ -199,6 +207,17 @@ function saveToPluginData(node: TextNode, stringResName: string) {
   );
 }
 
+function saveExtras(node: TextNode, extras?: StringResourceExtras) {
+  if (extras) {
+    node.setPluginData(
```

Comment:
Can you remind me what the reason is that the other data is saved in shared plugin data, but this one is in private data?

Reply:
This one is not used by the DesignCompose, only used by the plugin. It does not have to be in the shared plugin data so when we fetch the file it can be faster/save some storage too.